tests/cryptomatte_tests.py

```diff
@@ -524,19 +524,27 @@ class CryptomatteSetup(unittest.TestCase):
     def setUp(self):
         self.output_file_name = os.path.join(tempfile.gettempdir(), "result.exr")
 
+
         ai.AiBegin()
-        ai.AiMsgSetConsoleFlags(ai.AI_LOG_NONE)
-        ai.AiMsgSetConsoleFlags(ai.AI_LOG_WARNINGS | ai.AI_LOG_ERRORS)
+        self.universe = ai.AiUniverse()
+
+        msg_flags = ai.AI_LOG_WARNINGS|ai.AI_LOG_ERRORS|ai.AI_LOG_PROGRESS
+        ai.AiMsgSetLogFileFlags(self.universe, msg_flags)
+        ai.AiMsgSetConsoleFlags(self.universe, msg_flags)
+        ai.AiMsgWarning("[Cryptomatte test suite] - Rendering with Python. ")
```

Comment:
If you added in info level messages, you wouldn't have to make this a warning, which it's not.